react-design/my-app/src/RotatingBanner.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import SequentialButton from './SequentialButton';
+import ItemName from './ItemName';
+import IndicatorButton from './IndicatorButton';
+
+export default function RotatingBanner(props) {
+  let [index, setIndex] = useState(0);
```

Comment:
I'd advise using `const` for values that don't change during a single function call.